tools/shared/src/main/scala/org/scalajs/core/tools/optimizer/LinkingUnit.scala

```diff
@@ -1,15 +1,51 @@
 package org.scalajs.core.tools.optimizer
 
 import org.scalajs.core.ir
-import ir.Infos
+import ir.{Definitions, Infos}
 
 final class LinkingUnit(
     val classDefs: List[LinkedClass],
     val infos: Map[String, Infos.ClassInfo],
     val isComplete: Boolean) {
 
+  import LinkingUnit._
+
+  lazy val globalInfo: GlobalInfo = {
+    infos.get(Definitions.ClassClass).fold {
```

Comment:
I did it like this because I anticipate the addition of `isInterfacesDataAccessed`, which will depend on whether `getInterfaces` is called. I'll keep it like this, I think.